packages/camera/camera/ios/Classes/FLTCam_Test.h

```diff
@@ -17,9 +17,25 @@
 /// A dictionary to retain all in-progress FLTSavePhotoDelegates. The key of the dictionary is the
 /// AVCapturePhotoSettings's uniqueID for each photo capture operation, and the value is the
 /// FLTSavePhotoDelegate that handles the result of each photo capture operation. Note that photo
-/// capture operations may overlap, so we have to keep track of multiple delegates in progress,
+/// capture operations may overlap, so FLTCam has to keep track of multiple delegates in progress,
 /// instead of just a single delegate reference.
 @property(readonly, nonatomic)
     NSMutableDictionary<NSNumber *, FLTSavePhotoDelegate *> *inProgressSavePhotoDelegates;
 
+/// Delegate callback when receiving a new video or audio sample.
+/// Exposed for unit tests.
+- (void)captureOutput:(AVCaptureOutput *)output
+    didOutputSampleBuffer:(CMSampleBufferRef)sampleBuffer
+           fromConnection:(AVCaptureConnection *)connection;
+
+/// Init function exposed for unit test.
```

Comment:
"Initializes a camera instance. Allows for injecting dependencies that are usually internal."